From acda9bfefbfc99c35a1173be314a9aaac2a69ca5 Mon Sep 17 00:00:00 2001
From: Avi Drissman <avi@chromium.org>
Date: Thu, 7 Jul 2022 20:54:55 +0000
Subject: [PATCH 070/110] Disable failing tests

In addition, do minor clang-tidy cleanup.

(cherry picked from commit b67e952ed000379dea9358963c120669b8b8fc8f)

Bug: 1322740, 1322741
Change-Id: I7ec4539b9c635a414dd67c841e76dfcbb2d13332
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3635141
Commit-Queue: Jayson Adams <shrike@chromium.org>
Auto-Submit: Avi Drissman <avi@chromium.org>
Reviewed-by: Jayson Adams <shrike@chromium.org>
Cr-Original-Commit-Position: refs/heads/main@{#1001187}
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3749640
Auto-Submit: Robert Liao <robliao@chromium.org>
Reviewed-by: ccameron chromium <ccameron@chromium.org>
Owners-Override: Robert Liao <robliao@google.com>
Commit-Queue: Robert Liao <robliao@chromium.org>
Cr-Commit-Position: refs/branch-heads/5005@{#1229}
Cr-Branched-From: 5b4d9450fee01f821b6400e947b3839727643a71-refs/heads/main@{#992738}
---
 ...pp_shim_menu_controller_mac_browsertest.mm | 28 +++++++++----------
 .../native_app_window_cocoa_browsertest.mm    | 10 ++++---
 2 files changed, 19 insertions(+), 19 deletions(-)

diff --git a/chrome/browser/ui/cocoa/apps/app_shim_menu_controller_mac_browsertest.mm b/chrome/browser/ui/cocoa/apps/app_shim_menu_controller_mac_browsertest.mm
index e8b593a946be..c1b3c42f7e70 100644
--- a/chrome/browser/ui/cocoa/apps/app_shim_menu_controller_mac_browsertest.mm
+++ b/chrome/browser/ui/cocoa/apps/app_shim_menu_controller_mac_browsertest.mm
@@ -31,21 +31,18 @@ namespace {
 
 class AppShimMenuControllerBrowserTest
     : public extensions::PlatformAppBrowserTest {
- protected:
-  // The apps that can be installed and launched by SetUpApps().
-  enum AvailableApps { PACKAGED_1 = 0x1, PACKAGED_2 = 0x2, HOSTED = 0x4 };
-
-  AppShimMenuControllerBrowserTest()
-      : app_1_(nullptr),
-        app_2_(nullptr),
-        hosted_app_(nullptr),
-        initial_menu_item_count_(0) {}
-
+ public:
   AppShimMenuControllerBrowserTest(const AppShimMenuControllerBrowserTest&) =
       delete;
   AppShimMenuControllerBrowserTest& operator=(
       const AppShimMenuControllerBrowserTest&) = delete;
 
+ protected:
+  // The apps that can be installed and launched by SetUpApps().
+  enum AvailableApps { PACKAGED_1 = 0x1, PACKAGED_2 = 0x2, HOSTED = 0x4 };
+
+  AppShimMenuControllerBrowserTest() = default;
+
   // Start testing apps and wait for them to launch. |flags| is a bitmask of
   // AvailableApps.
   void SetUpApps(int flags) {
@@ -126,10 +123,10 @@ class AppShimMenuControllerBrowserTest
     return window_list.front();
   }
 
-  const extensions::Extension* app_1_;
-  const extensions::Extension* app_2_;
-  const extensions::Extension* hosted_app_;
-  NSUInteger initial_menu_item_count_;
+  const extensions::Extension* app_1_ = nullptr;
+  const extensions::Extension* app_2_ = nullptr;
+  const extensions::Extension* hosted_app_ = nullptr;
+  NSUInteger initial_menu_item_count_ = 0;
 };
 
 // Test that focusing an app window changes the menu bar.
@@ -181,8 +178,9 @@ IN_PROC_BROWSER_TEST_F(AppShimMenuControllerBrowserTest,
 }
 
 // Test that closing windows without main status do not update the menu.
+// Disabled; https://crbug.com/1322740.
 IN_PROC_BROWSER_TEST_F(AppShimMenuControllerBrowserTest,
-                       ClosingBackgroundWindowLeavesMenuBar) {
+                       DISABLED_ClosingBackgroundWindowLeavesMenuBar) {
   // Start with app1 active.
   SetUpApps(PACKAGED_1);
   extensions::AppWindow* app_1_app_window = FirstWindowForApp(app_1_);
diff --git a/chrome/browser/ui/cocoa/apps/native_app_window_cocoa_browsertest.mm b/chrome/browser/ui/cocoa/apps/native_app_window_cocoa_browsertest.mm
index 8b8712245076..50a5ea46a8d9 100644
--- a/chrome/browser/ui/cocoa/apps/native_app_window_cocoa_browsertest.mm
+++ b/chrome/browser/ui/cocoa/apps/native_app_window_cocoa_browsertest.mm
@@ -51,14 +51,15 @@ bool IsNSWindowFloating(NSWindow* window) {
 }
 
 class NativeAppWindowCocoaBrowserTest : public PlatformAppBrowserTest {
- protected:
-  NativeAppWindowCocoaBrowserTest() {}
-
+ public:
   NativeAppWindowCocoaBrowserTest(const NativeAppWindowCocoaBrowserTest&) =
       delete;
   NativeAppWindowCocoaBrowserTest& operator=(
       const NativeAppWindowCocoaBrowserTest&) = delete;
 
+ protected:
+  NativeAppWindowCocoaBrowserTest() = default;
+
   void SetUpAppWithWindows(int num_windows) {
     app_ = InstallExtension(
         test_data_dir_.AppendASCII("platform_apps").AppendASCII("minimal"), 1);
@@ -590,7 +591,8 @@ NSBitmapImageRep* ScreenshotNSWindow(NSWindow* window) {
 }  // namespace
 
 // Test that the colored frames have the correct color when active and inactive.
-IN_PROC_BROWSER_TEST_F(NativeAppWindowCocoaBrowserTest, FrameColor) {
+// Disabled; https://crbug.com/1322741.
+IN_PROC_BROWSER_TEST_F(NativeAppWindowCocoaBrowserTest, DISABLED_FrameColor) {
   // The hex values indicate an RGB color. When we get the NSColor later, the
   // components are CGFloats in the range [0, 1].
   extensions::AppWindow* app_window = CreateTestAppWindow(
-- 
2.38.1

